iam-login-service/src/main/java/it/infn/mw/iam/authn/multi_factor_authentication/IamAuthenticationMethodReference.java

```diff
@@ -27,7 +27,7 @@ public enum AuthenticationMethodReferenceValues {
     // Consult here for standardised reference values -
     // https://datatracker.ietf.org/doc/html/rfc8176
```

Comment:
Reading this RFC, we should probably not use "x509" (because it doesn't exist) and replace this string with "swk" which means "Proof-of-Possession (PoP) of a software-secured key". In fact, it refers to RFC4211 https://datatracker.ietf.org/doc/html/rfc4211